NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# Create A New Category

#### Changes Made
1. Added file `addCategory.js` to `client` directory.
1. Modified file `CategoryRepository.cs` to include `public void AddCategory(Category category)` and functionality.
2. Modified file `ICategoryRepository.cs` to include `public void AddCategory(Category category)` signature.
3. Modified file `CategoryController.cs` to include `public IActionResult Post (Category category)` and functionality.
4. Modified file `applicationViews.js` to include `route for new category form` and functionality.
5. Modified file `Category.js` to include `button to create new category` and functionality.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout iw-createCategory
    ```
6. Open a new Terminal tab (⌘T) and navigate to the server directory.
7. Test app functionality.
    > when viewing list of categories (after selecting categories nav option),
    >At the bottom of the categories list you should see a make new category button
    >Click on this button, and you will be directed to a form to create a new  category
    >On this page, enter in the name for the new category
    >Then click save, you should then be redirected to the list of categories
    > Observe your newly created category

8. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
